daemon/depend_filter.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+# -*- encoding: utf8 -*-
+
+#######################################################################################
+#
+# Copyright (C) 2012 - 2018 Jolla Ltd.
+#
+# Author: Simo Piiroinen <simo.piiroinen@jollamobile.com>
+#
+# All rights reserved.
+#
+# This file is part of nemo-keepalive package.
```

Comment:
Hmm?

Reply:
Well, it is from there... was going to let it be as it is since it is not a part of what gets built.

Reply:
But also it does not need to be mentioned.